test/data/err_999.py

```diff
@@ -0,0 +1,70 @@
+class torch:
```

Comment:
These tests look fine, though you will need to generate the expected error code data using `make update-tests`. This should create a corresponding `err_999.txt` file with the errors.